wsd/ProofKey.cpp

```diff
@@ -185,7 +185,10 @@ std::vector<unsigned char> Proof::RSA2CapiBlob(const std::vector<unsigned char>&
         throw ParseError("Proof key public exponent is longer than 4 bytes.");
     // make sure exponent length is correct; assume we are passed big-endian vectors
     std::vector<unsigned char> exponent32LE(4);
-    std::copy(exponent.rbegin(), exponent.rend(), exponent32LE.begin());
```

Comment:
Could you please also check if

    std::reverse_copy(exponent.begin(), exponent.end(), exponent32LE.begin());

makes a difference?

Reply:
Can try, but I doubt it will make a difference (because I agree with your next comment).

Reply:
This didn't work. I got this error:
```
/usr/include/c++/12/bits/stl_algo.h:1169:21: error: writing 1 byte into a region of size 0 [-Werror=stringop-overflow=]
 1169 |           *__result = *__last;
      |           ~~~~~~~~~~^~~~~~~~~
```